services/collaboration/README.md

```diff
@@ -0,0 +1,25 @@
+# Collaboration
+
+The collaboration service connects ocis with document servers such as collabora and onlyoffice using the WOPI protocol.
+
+Since this service requires an external service (onlyoffice, for example), it won't run by default with the general `ocis server` command. You need to run it manually with the `ocis collaboration server` command.
+
+## Requirements
+
+The collaboration service requires the target document server (onlyoffice, collabora, etc) to be up and running.
+Some oCIS services are also required to be running in order to register the GRPC service for the "open in app" action:
+* reva's gateway
+* app provider
```

Comment:
```suggestion
The following oCIS services are required to be running for the collaboration service in order to register the GRPC service for the `open in app` action:
* gateway
* app-provider
```
There is no `reva's gateway`but a gateway service. Also correcting the name of the app-provider service.

Reply:
It refers to the gateway API client in https://github.com/owncloud/ocis/pull/8374/files#diff-f02696d0615274e7591a84f42effbb7ab340a9b5c66976fe7f8a5ee8b5c80d92 . Since the code comes from the cs3org, I assumed we'd be connecting to a reva's gateway, but I could be wrong. Maybe @butonic can clarify.